docs/utilities.rst

```diff
@@ -1286,6 +1286,19 @@ hexidecimal digits.
     '0001..1e1f'
 
 
+``humanize_ipfs_uri(string)`` -> string
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
I think this works but maybe go ahead and extend the `^^^^^` to match line length of the header text.